studio/pages/new/[slug].tsx

```diff
@@ -242,112 +308,196 @@ export const Wizard = () => {
+                        <div
+                          style={{
+                            width: (PASSWORD_STRENGTH_PERCENTAGE as any)[passwordStrengthScore],
+                          }}
+                          className={`relative h-2 w-full ${
+                            (PASSWORD_STRENGTH_COLOR as any)[passwordStrengthScore]
+                          } transition-all duration-500 ease-out shadow-inner`}
+                        ></div>
+                      </div>
+                    )}
+                    <span
+                      className={
+                        passwordStrengthScore >= DEFAULT_MINIMUM_PASSWORD_STRENGTH
+                          ? 'text-green-600'
+                          : ''
+                      }
+                    >
+                      {passwordStrengthMessage
+                        ? passwordStrengthMessage
+                        : 'This is the password to your postgres database, so it must be a strong password and hard to guess.'}
+                    </span>
+                  </>
+                }
+                errorMessage={
+                  passwordStrengthWarning
+                    ? `${passwordStrengthWarning}. ${passwordErrorMessage}.`
+                    : passwordErrorMessage
+                }
+              />
+            </Panel.Content>
+
+            <Panel.Content className="Form section-block--body has-inputs-centered border-b border-panel-border-interior-light dark:border-panel-border-interior-dark">
+              <FormField
+                // @ts-ignore
+                label="Region"
+                type="select"
+                choices={REGIONS}
+                value={dbRegion}
+                onChange={onDbRegionChange}
+                description="Select a region close to you for the best performance."
+              />
+            </Panel.Content>
+
+            <Panel.Content className="Form section-block--body has-inputs-centered ">
+              <Listbox
+                label="Pricing Plan"
+                layout="horizontal"
+                value={dbPricingPlan}
+                onChange={onDbPricingPlanChange}
+                // @ts-ignore
+                descriptionText={
+                  <>
+                    Select a plan that suits your needs.&nbsp;
+                    <a className="underline" target="_blank" href="https://supabase.com/pricing">
+                      More details
+                    </a>
+                  </>
+                }
+              >
+                {Object.entries(PRICING_PLANS).map(([k, v]) => (
+                  <Listbox.Option
+                    key={k}
+                    label={v}
+                    value={v}
+                    addOnBefore={() => <IconDollarSign />}
+                  >
+                    {v}
+                  </Listbox.Option>
+                ))}
+              </Listbox>
+
+              <FreeProjectLimitWarning dbPricingPlan={dbPricingPlan} />
```

Comment:
could we bring the render conditions here instead of within the component? imo its clearer

`{isSelectFreeTier && isOverFreeProjectLimit && <FreeProjectLimitWarning />}`